04-cs-fundamentals/classroom/maths-exercises/essential-maths-exercises.md

```diff
@@ -0,0 +1,282 @@
+# Essential Mathematics - practice problems
+_CS fundamentals 0_
+
+## Overview
+Complete each problem manually (i.e. using paper and pencil) and tally your solution with the one provided for each problem below.
```

Comment:
I actually don't really know what this direction is -- what is the tally part for? And when do you tally your solution? Do you mean "check your solution agains the one provided for each problem below"?

Reply:
Oh yes, that's what I meant! Thank you, Dee! :-)